nestkernel/parameter.cpp

```diff
@@ -75,7 +76,7 @@ Parameter::apply( const NodeCollectionPTR& nc, const TokenArray& token_array )
           target_pos.size(),
           source_pos.size() ) );
     }
-    auto value = this->value( rng, source_pos, target_pos, *source_layer.get() );
+    auto value = this->value( rng, source_pos, target_pos, *source_layer.get(), nullptr );
```

Comment:
Passing `nullptr` here is not very elegant. Could we make `nullptr` a default value for the argument, so we wouldn't have to pass it here?

Reply:
We could, but changing `value( RngPtr, Node* )` to ` value( RngPtr, Node* node = nullptr )` would give a lot of warnings because of unused arguments.